src/id_scope.py

```diff
@@ -0,0 +1,42 @@
+import pygame
+
+from src.landing_game_object import LandingGameObject
+
+
+class IDSCOPE:
+    def __init__(self, object_list: pygame.sprite.Group):
+        self.object_list = object_list
+        self.standard_ids = {}
```

Comment:
Took me a little while to understand what it this for. Suggestion: self.reserved_ids

Reply:
Still not sure whether I like it here, or we should leave it to the caller